test/e2e/kubectl.go

```diff
@@ -202,8 +204,144 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl api-versions", func() {
+		It("should check if v1 is in available api versions", func() {
+			By("validating api verions")
+			output := runKubectl("api-versions")
+			if !strings.Contains(output, "Available Server Api Versions:") {
+				Failf("Missing caption in kubectl api-versions")
+			}
+			if !strings.Contains(output, "v1") {
+				Failf("No v1 in kubectl api-versions")
+			}
+		})
+	})
+
+	Describe("Kubectl cluster-info", func() {
+		It("should check if Kubernetes master services is included in cluster-info", func() {
+			By("validating cluster-info")
+			output := runKubectl("cluster-info")
+			// Can't check exact strings due to terminal controll commands (colors)
+			requiredItems := []string{"Kubernetes master", "KubeDNS", "Heapster", "is running at"}
```

Comment:
I think the only one we can require is `Kubernetes master`. KubeDNS and Heapster are optional, no? If we can assume they are required at least for gce/gke, then this test should include a `SkipUnlessProviderIs("gce", "gke")`